classes/class-twenty-twenty-one-customize.php

```diff
@@ -81,6 +81,40 @@ public function register( $wp_customize ) {
 				)
 			);
 
+			/**
+			 * Add excerpt or full text selector to customizer
+			 */
+
+			$wp_customize->add_section(
+				'theme_settings',
+				array(
+					'title'    => __( 'Theme settings', 'twentytwentyone' ),
```

Comment:
Can we use `esc_html__` instead of `__` here?
Since this will be a core theme everything needs to be escaped

```suggestion
					'title'    => esc_html__( 'Theme settings', 'twentytwentyone' ),
```